composer.json

```diff
@@ -36,18 +36,15 @@
     "bin": ["bin/check_perms"],
     "require": {
         "symfony/polyfill-iconv": "^1",
-        "phpro/grumphp": "^1",
+        "phpro/grumphp": "^2.5",
         "drupal/coder": "^8",
         "phpcompatibility/php-compatibility": "^9.3",
         "pheromone/phpcs-security-audit": "^2.0",
         "squizlabs/php_codesniffer": "^3.4",
         "dealerdirect/phpcodesniffer-composer-installer": "*",
         "mglaman/phpstan-drupal": "^1.1",
         "phpstan/extension-installer": "^1.1",
-        "phpstan/phpstan-deprecation-rules": "^1.0",
-        "vimeo/psalm": "^4",
```

Comment:
GrumPHP 2.x needs amphp/amp: ^3.0, vimeo/psalm 4 and 5 require amphp/amp: ^2.0. Only resolution is to remove Psalm. Psalm 6 dev seems to be on amphp/amp: ^3.0 https://packagist.org/packages/vimeo/psalm#dev-master